benchmark_utils/backbones_architecture.py

```diff
@@ -33,6 +34,25 @@ def forward(self, x, sample_weight=None):
 
         return x
 
+class OfficeConvNet(nn.Module):
+    def __init__(self, n_classes):
+        super().__init__()
+        
+        # Load pretrained ResNet50
+        self.resnet = resnet50(weights=ResNet50_Weights.DEFAULT)
+        
+        num_ftrs = self.resnet.fc.in_features
+        self.fc1 = nn.Linear(num_ftrs, 1024)
+        self.fc2 = nn.Linear(1024, n_classes)
+        
+        # Replace ResNet's fc layer with Identity
+        self.resnet.fc = nn.Identity()
+
+    def forward(self, x, sample_weight=None):
+        x = self.resnet(x)
+        x = F.relu(self.fc1(x))
+        x = self.fc2(x)
+        return x
```

Comment:
An overall comment: did you see a paper applying this architecture to Officehome? Maybe a Resnet18 is enough?